portal/values.yaml

```diff
@@ -27,7 +28,7 @@ frontend:
   service:
     name: http
     type: NodePort
-    externalPort: 30766
+    externalPort: 30767
```

Comment:
Any reason for the port change? might mean asking a different URL to be whitelisted at EBI Portal...

Reply:
We are already using the port 30767 on portaldev.

Reply:
It is more practical. We are actually using the 30767 port on portaldev and it is also the port allowed for development purposes on minikube. The main installation of the portal can be configured to use the 30766 port using the config file.